packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1082,10 +1095,11 @@ export class ArcRotateCamera extends TargetCamera {
      * @param toBoundingCenter In case of a mesh target, defines whether to target the mesh position or its bounding information center
      * @param allowSamePosition If false, prevents reapplying the new computed position if it is identical to the current one (optim)
      * @param cloneAlphaBetaRadius If true, replicate the current setup (alpha, beta, radius) on the new target
+     * @param setManually Defines the target is Automatic calculation through the program or Setting by the user
      */
-    public setTarget(target: AbstractMesh | Vector3, toBoundingCenter = false, allowSamePosition = false, cloneAlphaBetaRadius = false): void {
+    public setTarget(target: AbstractMesh | Vector3, toBoundingCenter = false, allowSamePosition = false, cloneAlphaBetaRadius = false, setManually = false): void {

```

Comment:
@sebavan @PolygonalSun
My previous idea was to change the _targetSetManually variable by adding a parameter to setTarget so that the user does not have to explicitly change the parameter
If you agree, I can resume the previous operation